@here/harp-mapview/lib/PickHandler.ts

```diff
@@ -147,8 +149,19 @@ export class PickHandler {
             this.mapView.textElementsRenderer.pickTextElements(scenePosition, pickResults);
         }
 
-        // calculate objects intersecting the picking ray
-        const intersects = rayCaster.intersectObjects(this.mapView.worldRootObject.children, true);
+        const intersects: THREE.Intersection[] = [];
```

Comment:
The `PickHandler` doesn't seem to have a unit test right now so we should probably add one.

Reply:
Agreed. But I'm gonna change this code further as part of the same ticket in next review and then I'm gonna delete PickHandler and move the code to PickingRaycaster. I'd rather add the tests then, otherwise I'd have to keep changing the tests on the way.
I've tested manually this change on planar/sphere and w/o terrain and it seems to be fine.